examples/fitness/apk/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
     <uses-permission android:name="org.domokit.sky.shell.permission.C2D_MESSAGE" />
     <!-- end for GCM -->
 
-    <application android:icon="@mipmap/ic_launcher" android:label="Fitness" android:name="org.domokit.fitness.FitnessApplication">
+    <application android:icon="@mipmap/ic_launcher" android:label="Fitness" android:name="org.domokit.sky.shell.SkyApplication">
```

Comment:
The only thing FitnessApplication did was register the GCM mojo service. This is now done by the gcm service itself via this new dynamic registration system.